docs/contributor/01-00-installation.md

```diff
@@ -0,0 +1,241 @@
+# Install the Istio module
+
+- [Install the Istio module](#install-the-istio-module)
+  - [Prerequisites](#prerequisites)
+  - [Install Istio Operator manually](#install-istio-operator-manually)
+    - [Use Istio Operator to install or uninstall Istio](#use-istio-operator-to-install-or-uninstall-istio)
+  - [Install Istio in modular Kyma on a local k3d cluster](#install-istio-in-modular-kyma-on-a-local-k3d-cluster)
+    - [Use Lifecycle Manager to install Istio in modular Kyma on k3d](#use-lifecycle-manager-to-install-istio-in-modular-kyma-on-k3d)
+  - [Install with artifacts built for the `main` branch of the Istio repository](#install-with-artifacts-built-for-the-main-branch-of-the-istio-repository)
+
+## Prerequisites
+
+- Access to a Kubernetes cluster (you can use [k3d](https://k3d.io/v5.5.1/))
+- [kubectl](https://kubernetes.io/docs/tasks/tools/)
+- [kubebuilder](https://book.kubebuilder.io/)
+- [Docker](https://www.docker.com)
+- [Kyma CLI](https://kyma-project.io/docs/kyma/latest/04-operation-guides/operations/01-install-kyma-CLI)
+
+## Install Istio Operator manually
+
+1. Clone the project.
+
+```bash
+git clone https://github.com/kyma-project/istio.git && cd istio
+```
+
+2. Set Istio Operator image name.
+
+```bash
+export IMG=istio-operator:0.0.1
+export K3D_CLUSTER_NAME=kyma
+```
+
+3. Provision k3d cluster.
+
+```bash
+kyma provision k3d
+```
+>**TIP:** To verify the correctness of the project, build it using the `make build` command.
+
+4. Build the image.
+
+```bash
+make docker-build
+```
+
+5. Push the image to the registry.
+
+<div tabs name="Push image" group="istio-operator-installation">
+  <details>
+  <summary label="k3d">
+  k3d
+  </summary>
+
+   ```bash
+   k3d image import $IMG -c $K3D_CLUSTER_NAME
+   ```
+
+  </details>
+  <details>
+  <summary label="Docker registry">
+  Globally available Docker registry
+  </summary>
+
+   ```bash
+   make docker-push
+   ```
+
+  </details>
+</div>
+
+6. Deploy.
```

Comment:
What do we deploy here? Istio Operator?
Maybe it's worth adding it here, just in case.